fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Alias.java

```diff
@@ -98,12 +98,11 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-        if (!super.equals(o)) {
-            return false;
-        }
-        Alias alias = (Alias) o;
-        return exprId.equals(alias.exprId) && name.equals(alias.name)
-                && qualifier.equals(alias.qualifier) && children.equals(alias.children);
+        Alias that = (Alias) o;
+        return exprId.equals(that.exprId)
```

Comment:
how about remove "exprId.equals(that.exprId)" ?
do you have an example to illustrate its necessity?